### PR TITLE
add resource type

### PR DESCRIPTION
**Pull request by bangnokia**
add resource type

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/php/php-data-types/index.md [hunk 1]
```diff
@@ -103,3 +103,23 @@ $herbie = new Car();
 echo $herbie->model;
 ?>
 ```
+
+### PHP Resource
+
+A resource is a special variable, holding a reference to an external resource. Resources are created and used by special functions. See the [appendix](http://php.net/manual/en/resource.php) for a listing of all these functions and the corresponding resource types. You can use [get_resource_type()](http://php.net/manual/en/function.get-resource-type.php) function to see resource type.
```
Comment (co16353sidak) [inline guide/english/php/php-data-types/index.md]: here in the links, there is no appendix in the file being edited over here, so simply writing the word `appendix` will make it unclear to the reader about where the link will go.
Instead write `See the [official PHP documentation] for a ...` to make it more readable.

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@bangnokia 
Please check comments for suggestions.

**Review by theNaut1lus — APPROVED** [review]
Changes made, simple merge.

@RandellDawson

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**